package.json

```diff
@@ -29,28 +29,28 @@
     "version": "standard-version",
     "reset-hard": "git clean -dfx && git reset --hard && npm i",
     "prepare-release": "run-s reset-hard test cov:check doc:html version doc:publish",
-    "generate-types": "typechain --target ethers-v5 --out-dir 'types/ethers-v5' './node_modules/@lukso/universalprofile-smart-contracts/build/contracts/*.json'"
+    "generate-types": "typechain --target ethers-v5 --out-dir 'types/ethers-v5' './node_modules/@lukso/universalprofile-smart-contracts/build/artifacts/*.json'"
   },
   "engines": {
     "node": ">=10"
   },
   "dependencies": {
     "@erc725/erc725.js": "^0.6.2-beta.4",
-    "@ethersproject/abi": "^5.4.0",
+    "@ethersproject/abi": "^5.4.1",
     "browser-image-compression": "^1.0.15",
-    "ethers": "5.4.5",
+    "ethers": "5.4.6",
     "ipfs-http-client": "52.0.2",
     "ipfs-utils": "^8.1.6"
   },
   "devDependencies": {
     "@ethereum-waffle/jest": "^3.3.0",
-    "@lukso/universalprofile-smart-contracts": "0.1.1",
+    "@lukso/universalprofile-smart-contracts": "^0.1.3",
```

Comment:
Do you want to use "^0.1.3"
Instead of "0.1.3" ?
Because we are still in early version maybe it might be more robust to set a specific version?

Reply:
```suggestion
    "@lukso/universalprofile-smart-contracts": "0.1.3",
```